marklogic-data-hub-central/src/main/java/com/marklogic/hub/central/HubCentral.java

```diff
@@ -42,23 +37,19 @@
     public void afterPropertiesSet() {
         logger.info("Will connect to MarkLogic host: " + host);
 
-        hubProject = new HubProjectImpl();
-        logger.info("Temporarily initializing project at: " + projectPath);
-        hubProject.createProject(projectPath);
-
-        HubConfigImpl tempHubConfig = new HubConfigImpl(hubProject, environment);
-        tempHubConfig.initHubProject();
-        logger.info("Initialized project at: " + getProjectDirectory());
-
         if (useLocalDefaults) {
             logger.info("Local defaults of digest authentication and no SSL will be used when connecting to MarkLogic");
         }
 
+        temporaryLoadDataDirectory = new File("build/data-sets");
```

Comment:
This is called "temporaryLoadDataDirectory". Do we have any intention of cleaning up the directory or are we saying that it is temporary, as in it will go away when we rework the "Load Data" feature?

Reply:
Right, it will go away once we tackle that set of stories.